src/docker/composebuilder/dependency_builder.go

```diff
@@ -50,14 +50,13 @@ func (g *LocalDependency) GetDockerConfig() (types.DockerConfig, error) {
 func (g *LocalDependency) GetServiceEnvVariables() map[string]string {
 	result := map[string]string{}
 	result[fmt.Sprintf("%s_HOST", strings.ToUpper(g.name))] = g.name
-	util.Merge(result, g.config.Config.ServiceEnvironment)
```

Comment:
Remove `ServiceEnvironment` as we can use app wide environment variables to achieve the same thing